.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Descriptive summary of request
+
+Include a detailed description of the documentation issue you are submitting. Include
+the version of software (Sufia, Hyrax, Hyku, etc.) that you are working in.
+
+If you are submitting an issue related to existing documentation, include the "Existing Documentation" section
+with link(s) to the document in question and describe the question that is incorrect. If possible, offer a suggested
```

Comment:
Instead of 'and describe the question that is incorrect', how about 'and describe how it is incorrect'